innative/instructions.cpp

```diff
@@ -701,6 +701,8 @@ IN_ERROR Compiler::CompileMemCopy(varuint32 dst_mem, varuint32 src_mem)
   return ERR_SUCCESS;
 }
 
+extern "C" int __favor;
```

Comment:
What is this doing here?

Reply:
whoops, that was just to figure out what value it should be :P